dags/pegs-challenge-submission-dag.py

```diff
@@ -1,26 +1,35 @@
+import uuid
+import os
 from datetime import datetime
+import pandas as pd
 
 from airflow.decorators import dag, task
 from airflow.models import Param
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
 
 from orca.services.nextflowtower import NextflowTowerHook
 from orca.services.nextflowtower.models import LaunchInfo
 from orca.services.synapse import SynapseHook
 
+REGION_NAME = "us-east-1"
+BUCKET_NAME = "pegs-challenge-project-tower-scratch"
+FILE_NAME = f"submissions_{uuid.uuid4()}.csv"
+KEY = "work"
```

Comment:
Using `work` is fine, but for my DAG I used `10days/<challenge_name>`. I liked this approach because then the manifests would be cleared after 10 days instead of 30 (the default for `scratch` bucket storage) and also it would be extra easy to find the manifests in one place if need be.